exchange/utils.go

```diff
@@ -192,7 +192,7 @@ func (rs *requestSplitter) cleanOpenRTBRequests(ctx context.Context,
 		auctionPermissions := gdprPerms.AuctionActivitiesAllowed(ctx, coreBidder, openrtb_ext.BidderName(bidder))
 
 		// privacy blocking
-		if rs.isBidderBlockedByPrivacy(req, auctionReq.Activities, auctionPermissions, coreBidder, openrtb_ext.BidderName(bidder)) {
+		if rs.isBidderBlockedByPrivacy(reqWrapperCopy, auctionReq.Activities, auctionPermissions, coreBidder, openrtb_ext.BidderName(bidder)) {
```

Comment:
Isn't this a change of signature which would require `isBidderBlockedByPrivacy()` to be updated?

Reply:
Ahh, the function has already been updated, its just getting the original request wrapper before this change.